src/legacy/core_plugins/kibana/public/dashboard/grid/dashboard_grid.tsx

```diff
@@ -117,10 +116,10 @@ const config = { monitorWidth: true };
 const ResponsiveSizedGrid = sizeMe(config)(ResponsiveGrid);
 
 interface Props extends ReactIntl.InjectedIntlProps {
-  panels: PanelStateMap;
+  panels: { [key: string]: SavedDashboardPanel };
   getEmbeddableFactory: (panelType: string) => EmbeddableFactory;
   dashboardViewMode: DashboardViewMode.EDIT | DashboardViewMode.VIEW;
-  onPanelsUpdated: (updatedPanels: PanelStateMap) => void;
+  onPanelsUpdated: (updatedPanels: { [key: string]: SavedDashboardPanel }) => void;
```

Comment:
nit: `{ [key: string]: SavedDashboardPanel }` seems to be duplicated several places in this PR. It's easy enough to write out, but I wonder if it's worth creating an interface for it?

Reply:
Yeaaa, so as you can see I originally had `PanelStateMap` and at some point I decided it was better for my brain to see the map definition than have it refactored away... so I switched it to the raw form.  But then I also realized, just how much it is used everywhere and started thinking, maybe I should have kept it.  :) . I'll add it back!